image/bmp.cpp

```diff
@@ -178,7 +178,7 @@ bool writeBMP(Common::WriteStream &out, const Graphics::Surface &input, const by
 	out.writeUint16LE(1);
 	out.writeUint16LE(24);
 	out.writeUint32LE(0);
-	out.writeUint32LE(0);
+	out.writeUint32LE(surface->h * dstPitch);
```

Comment:
This is the size of the raw bitmap data; a dummy 0 can be given for BI_RGB (uncompressed) bitmaps.

Reference for BITMAPINFOHEADER:
https://en.wikipedia.org/wiki/BMP_file_format#DIB_header_(bitmap_information_header)

Reply:
I added it for consistency with the old Wintermute code, but I can remove it if you'd prefer.

Reply:
This field data was added by @somaen in commit 764b7edee4b05804ef42c5991b450f606afeaaab.
It was added for debugging. I believe we can safely revert that, and follow the BMP spec.

Reply:
OK, done.